scripts/e2e-test.sh

```diff
@@ -50,6 +53,7 @@ while [ "$#" -gt 0 ]; do
       exit 1
       ;;
   esac
+  shift
```

Comment:
that assumes that all arguments will always have a parameter. For example the new option --long that I'm introducing does not have any parameter. So I would need to insert all shifts back then.

Reply:
sorry, ignore my comment. as discussed in the chat it will work.